server/workers/openDataCaller.js

```diff
@@ -8,7 +8,7 @@ category!="NON-CRIMINAL" AND \
 category!="BAD CHECKS" AND \
 category!="BRIBERY" AND \
 category!="FORGERY/COUNTERFEITING" AND \
-category!="FRAUD"&$limit=30000';
+category!="FRAUD"&$limit=12000';
```

Comment:
I assume you checked that setting the limit to 12,000 gives us the most recent 12,000 reports, as opposed to the first 12,000 or a random selection of 12,000? I am gonna merge this b/c I want to work with the smaller data set but let me know!